packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -188,9 +188,11 @@ export interface ParseCfnOptions {
  */
 export class CfnParser {
   private readonly options: ParseCfnOptions;
+  public readonly parameterValues?: { [parameterName: string]: any };
```

Comment:
Thoughts on just leaving `parameterValues` as a member of options (renamed to `parameters`) and accessing it as `this.options.parameters`? Doing it like this would mandate the use of `|| {}` and `!` throughout this class.